Google.PowerShell/Provider/CacheItem.cs

```diff
@@ -37,5 +37,13 @@ public CacheItem(Func<T> update, TimeSpan cacheLifetime)
             _update = update;
             _cacheLifetime = cacheLifetime;
         }
+
+        /// <summary>
+        /// Forces the CacheItem to do a new update the next time the value is requested.
+        /// </summary>
+        public void Obsolete()
```

Comment:
`Obsolete` sounds a bit more like an adjective/property to me. Perhaps `Invalidate` or `ForceRefresh` would be more appropriate?